io/net/virt-net/network_virtualization.py

```diff
@@ -127,6 +127,7 @@ def setUp(self):
         self.host_public_ip = self.params.get('host_public_ip', default=None)
         self.host_user = self.params.get('user_name', default=None)
         self.host_password = self.params.get('host_password', default=None)
+        self.is_mlx_driver = self.params.get('is_mlx_driver', default=True)
```

Comment:
@vaishnavibhat can we add this validation in the script itself? just to reduce the number of input parameters.

Reply:
> @vaishnavibhat can we add this validation in the script itself? just to reduce the number of input parameters.

We have logic to find the underlying driver with the proc file system. In this case we are able to find out 'ibmvnic' as driver. But to see if its mellanox or non-mellanox adapter we do not have any logic. Hence the yaml parameter.

Reply:
we do not have the adapter listed in lpar to verify and also this is virtual driver test.. so unaware of under laying adapter

Reply:
ok